### PR TITLE
Setup rakefile and simplecov

### PR DESCRIPTION
**Pull request by tylertomlinson**
Created Test_helper.rb with SimpleCov
Created Rakefile.rb

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "simplecov"
+
+SimpleCov.start do
+end
+
```
Comment (Maxwell-Baird) [inline test/test_helper.rb:5]: Make sure to add the test later on

File: test/test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+require "minitest/autorun"
```
Comment (whitneykidd) [inline test/test_helper.rb:6]: Nice, I like the idea of requiring these here

File: test/test_helper.rb (hunk 1 above)
Comment (tylertomlinson) [reply test/test_helper.rb:5]: Yes. require_relative 'test_helper' is now life.

File: Rakefile.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'rake/testtask'
+
+task default: %w[test]
+
+Rake::TestTask.new do |task|
```
Comment (alex-latham) [inline Rakefile.rb:5]: This is new to me!

**Pull request merged**